### ISSUE TITLE
fish support

### ISSUE THREAD

**Issue by srid** [wting/autojump#128]
i can haz [fish](http://ridiculousfish.com/shell/) support?

**Comment by BenShaw** [comment]
Im having issues getting autojump working in fish shell
Installed from apt im on version 2.6
Ran autojump and i get a message to read the config
Config does not have instructions for fish
Can someone confirm the line i need to add to my fish config please

**Comment by frederickjh** [comment]
@BenShaw  I copied autojump.fish to ~/.config/fish/functions/ on Ubuntu Linux and it worked. Not sure what OS you are using Fish on. YMMV

**Comment by frederickjh** [comment]
I ended up uninstalling autojump and installing [z-fish](https://github.com/sjl/z-fish) like @espen. I did not like the fact that every time cd or jumped my fish prompt was showing a still running process.
![selection_548](https://user-images.githubusercontent.com/864159/34502912-17e17826-f016-11e7-935a-368273ca32b2.png)
I also prefer z.fish's frequency rating. It is more what I would expect and matches what I have come to expect in fish shell.
